diff --git a/console/core/index.ts b/console/core/index.ts
index 3d4e4be..2f15c42 100644
--- a/console/core/index.ts
+++ b/console/core/index.ts
@@ -1,6 +1,6 @@
-import { K8sModel as K8sKind } from '@console/dynamic-plugin-sdk/src/api/common-types';
+import { K8sModel } from '@openshift-console/dynamic-plugin-sdk/lib/api/common-types';
 
-export const PrometheusModel: K8sKind = {
+export const PrometheusModel: K8sModel = {
   kind: 'Prometheus',
   label: 'Prometheus',
   // t('public~Prometheus')
@@ -17,7 +17,7 @@ export const PrometheusModel: K8sKind = {
   propagationPolicy: 'Foreground',
 };
 
-export const ServiceMonitorModel: K8sKind = {
+export const ServiceMonitorModel: K8sModel = {
   kind: 'ServiceMonitor',
   label: 'ServiceMonitor',
   // t('public~ServiceMonitor')
@@ -34,7 +34,7 @@ export const ServiceMonitorModel: K8sKind = {
   propagationPolicy: 'Foreground',
 };
 
-export const AlertmanagerModel: K8sKind = {
+export const AlertmanagerModel: K8sModel = {
   kind: 'Alertmanager',
   label: 'Alertmanager',
   // t('public~Alertmanager')
@@ -51,7 +51,7 @@ export const AlertmanagerModel: K8sKind = {
   propagationPolicy: 'Foreground',
 };
 
-export const ServiceModel: K8sKind = {
+export const ServiceModel: K8sModel = {
   apiVersion: 'v1',
   label: 'Service',
   // t('public~Service')
@@ -66,7 +66,7 @@ export const ServiceModel: K8sKind = {
   labelPluralKey: 'public~Services',
 };
 
-export const PodModel: K8sKind = {
+export const PodModel: K8sModel = {
   apiVersion: 'v1',
   label: 'Pod',
   // t('public~Pod')
@@ -81,7 +81,7 @@ export const PodModel: K8sKind = {
   labelPluralKey: 'public~Pods',
 };
 
-export const ContainerModel: K8sKind = {
+export const ContainerModel: K8sModel = {
   apiVersion: 'v1',
   label: 'Container',
   // t('public~Container')
@@ -95,7 +95,7 @@ export const ContainerModel: K8sKind = {
   labelPluralKey: 'public~Containers',
 };
 
-export const DaemonSetModel: K8sKind = {
+export const DaemonSetModel: K8sModel = {
   label: 'DaemonSet',
   // t('public~DaemonSet')
   labelKey: 'public~DaemonSet',
@@ -112,7 +112,7 @@ export const DaemonSetModel: K8sKind = {
   labelPluralKey: 'public~DaemonSets',
 };
 
-export const ReplicationControllerModel: K8sKind = {
+export const ReplicationControllerModel: K8sModel = {
   apiVersion: 'v1',
   label: 'ReplicationController',
   // t('public~ReplicationController')
@@ -128,7 +128,7 @@ export const ReplicationControllerModel: K8sKind = {
   labelPluralKey: 'public~ReplicationControllers',
 };
 
-export const HorizontalPodAutoscalerModel: K8sKind = {
+export const HorizontalPodAutoscalerModel: K8sModel = {
   label: 'HorizontalPodAutoscaler',
   // t('public~HorizontalPodAutoscaler')
   labelKey: 'public~HorizontalPodAutoscaler',
@@ -144,7 +144,7 @@ export const HorizontalPodAutoscalerModel: K8sKind = {
   labelPluralKey: 'public~HorizontalPodAutoscalers',
 };
 
-export const ServiceAccountModel: K8sKind = {
+export const ServiceAccountModel: K8sModel = {
   apiVersion: 'v1',
   label: 'ServiceAccount',
   // t('public~ServiceAccount')
@@ -159,7 +159,7 @@ export const ServiceAccountModel: K8sKind = {
   labelPluralKey: 'public~ServiceAccounts',
 };
 
-export const ReplicaSetModel: K8sKind = {
+export const ReplicaSetModel: K8sModel = {
   label: 'ReplicaSet',
   // t('public~ReplicaSet')
   labelKey: 'public~ReplicaSet',
@@ -176,7 +176,7 @@ export const ReplicaSetModel: K8sKind = {
   labelPluralKey: 'public~ReplicaSets',
 };
 
-export const DeploymentModel: K8sKind = {
+export const DeploymentModel: K8sModel = {
   label: 'Deployment',
   // t('public~Deployment')
   labelKey: 'public~Deployment',
@@ -193,7 +193,7 @@ export const DeploymentModel: K8sKind = {
   labelPluralKey: 'public~Deployments',
 };
 
-export const DeploymentConfigModel: K8sKind = {
+export const DeploymentConfigModel: K8sModel = {
   label: 'DeploymentConfig',
   // t('public~DeploymentConfig')
   labelKey: 'public~DeploymentConfig',
@@ -210,7 +210,7 @@ export const DeploymentConfigModel: K8sKind = {
   labelPluralKey: 'public~DeploymentConfigs',
 };
 
-export const BuildConfigModel: K8sKind = {
+export const BuildConfigModel: K8sModel = {
   label: 'BuildConfig',
   // t('public~BuildConfig')
   labelKey: 'public~BuildConfig',
@@ -227,7 +227,7 @@ export const BuildConfigModel: K8sKind = {
   labelPluralKey: 'public~BuildConfigs',
 };
 
-export const BuildModel: K8sKind = {
+export const BuildModel: K8sModel = {
   label: 'Build',
   // t('public~Build')
   labelKey: 'public~Build',
@@ -244,7 +244,7 @@ export const BuildModel: K8sKind = {
   labelPluralKey: 'public~Builds',
 };
 
-export const TemplateModel: K8sKind = {
+export const TemplateModel: K8sModel = {
   label: 'Template',
   // t('public~Template')
   labelKey: 'public~Template',
@@ -261,7 +261,7 @@ export const TemplateModel: K8sKind = {
   labelPluralKey: 'public~Templates',
 };
 
-export const TemplateInstanceModel: K8sKind = {
+export const TemplateInstanceModel: K8sModel = {
   label: 'Template Instance',
   apiVersion: 'v1',
   apiGroup: 'template.openshift.io',
@@ -273,7 +273,7 @@ export const TemplateInstanceModel: K8sKind = {
   labelPlural: 'Template Instances',
 };
 
-export const ImageStreamModel: K8sKind = {
+export const ImageStreamModel: K8sModel = {
   label: 'ImageStream',
   // t('public~ImageStream')
   labelKey: 'public~ImageStream',
@@ -290,7 +290,7 @@ export const ImageStreamModel: K8sKind = {
   labelPluralKey: 'public~ImageStreams',
 };
 
-export const ImageStreamTagModel: K8sKind = {
+export const ImageStreamTagModel: K8sModel = {
   label: 'ImageStreamTag',
   // t('public~ImageStreamTag')
   labelKey: 'public~ImageStreamTag',
@@ -307,7 +307,7 @@ export const ImageStreamTagModel: K8sKind = {
   labelPluralKey: 'public~ImageStreamTags',
 };
 
-export const ImageStreamImportsModel: K8sKind = {
+export const ImageStreamImportsModel: K8sModel = {
   label: 'ImageStreamImport',
   // t('public~ImageStreamImport')
   labelKey: 'public~ImageStreamImport',
@@ -323,7 +323,7 @@ export const ImageStreamImportsModel: K8sKind = {
   labelPluralKey: 'ImageStreamImports',
 };
 
-export const JobModel: K8sKind = {
+export const JobModel: K8sModel = {
   label: 'Job',
   // t('public~Job')
   labelKey: 'public~Job',
@@ -340,7 +340,7 @@ export const JobModel: K8sKind = {
   labelPluralKey: 'public~Jobs',
 };
 
-export const NodeModel: K8sKind = {
+export const NodeModel: K8sModel = {
   apiVersion: 'v1',
   label: 'Node',
   // t('public~Node')
@@ -354,7 +354,7 @@ export const NodeModel: K8sKind = {
   labelPluralKey: 'public~Nodes',
 };
 
-export const CertificateSigningRequestModel: K8sKind = {
+export const CertificateSigningRequestModel: K8sModel = {
   apiVersion: 'v1',
   apiGroup: 'certificates.k8s.io',
   label: 'CertificateSigningRequest',
@@ -369,7 +369,7 @@ export const CertificateSigningRequestModel: K8sKind = {
   labelPluralKey: 'public~CertificateSigningRequests',
 };
 
-export const EventModel: K8sKind = {
+export const EventModel: K8sModel = {
   apiVersion: 'v1',
   label: 'Event',
   // t('public~Event')
@@ -384,7 +384,7 @@ export const EventModel: K8sKind = {
   labelPluralKey: 'public~Events',
 };
 
-export const ComponentStatusModel: K8sKind = {
+export const ComponentStatusModel: K8sModel = {
   apiVersion: 'v1',
   label: 'ComponentStatus',
   // t('public~ComponentStatus')
@@ -398,7 +398,7 @@ export const ComponentStatusModel: K8sKind = {
   id: 'componentstatus',
 };
 
-export const NamespaceModel: K8sKind = {
+export const NamespaceModel: K8sModel = {
   apiVersion: 'v1',
   label: 'Namespace',
   // t('public~Namespace')
@@ -412,7 +412,7 @@ export const NamespaceModel: K8sKind = {
   labelPluralKey: 'public~Namespaces',
 };
 
-export const ProjectModel: K8sKind = {
+export const ProjectModel: K8sModel = {
   apiVersion: 'v1',
   apiGroup: 'project.openshift.io',
   label: 'Project',
@@ -427,7 +427,7 @@ export const ProjectModel: K8sKind = {
   labelPluralKey: 'public~Projects',
 };
 
-export const ProjectRequestModel: K8sKind = {
+export const ProjectRequestModel: K8sModel = {
   apiVersion: 'v1',
   apiGroup: 'project.openshift.io',
   label: 'ProjectRequest',
@@ -442,7 +442,7 @@ export const ProjectRequestModel: K8sKind = {
   labelPluralKey: 'public~ProjectRequests',
 };
 
-export const IngressModel: K8sKind = {
+export const IngressModel: K8sModel = {
   label: 'Ingress',
   // t('public~Ingress')
   labelKey: 'public~Ingress',
@@ -458,7 +458,7 @@ export const IngressModel: K8sKind = {
   id: 'ingress',
 };
 
-export const RouteModel: K8sKind = {
+export const RouteModel: K8sModel = {
   label: 'Route',
   // t('public~Route')
   labelKey: 'public~Route',
@@ -474,7 +474,7 @@ export const RouteModel: K8sKind = {
   id: 'route',
 };
 
-export const ConfigMapModel: K8sKind = {
+export const ConfigMapModel: K8sModel = {
   apiVersion: 'v1',
   label: 'ConfigMap',
   // t('public~ConfigMap')
@@ -489,7 +489,7 @@ export const ConfigMapModel: K8sKind = {
   labelPluralKey: 'public~ConfigMaps',
 };
 
-export const SecretModel: K8sKind = {
+export const SecretModel: K8sModel = {
   apiVersion: 'v1',
   label: 'Secret',
   // t('public~Secret')
@@ -504,7 +504,7 @@ export const SecretModel: K8sKind = {
   labelPluralKey: 'public~Secrets',
 };
 
-export const ClusterRoleBindingModel: K8sKind = {
+export const ClusterRoleBindingModel: K8sModel = {
   label: 'ClusterRoleBinding',
   // t('public~ClusterRoleBinding')
   labelKey: 'public~ClusterRoleBinding',
@@ -519,7 +519,7 @@ export const ClusterRoleBindingModel: K8sKind = {
   labelPluralKey: 'public~ClusterRoleBindings',
 };
 
-export const ClusterRoleModel: K8sKind = {
+export const ClusterRoleModel: K8sModel = {
   label: 'ClusterRole',
   // t('public~ClusterRole')
   labelKey: 'public~ClusterRole',
@@ -534,7 +534,7 @@ export const ClusterRoleModel: K8sKind = {
   labelPluralKey: 'public~ClusterRoles',
 };
 
-export const RoleBindingModel: K8sKind = {
+export const RoleBindingModel: K8sModel = {
   label: 'RoleBinding',
   // t('public~RoleBinding')
   labelKey: 'public~RoleBinding',
@@ -550,7 +550,7 @@ export const RoleBindingModel: K8sKind = {
   labelPluralKey: 'public~RoleBindings',
 };
 
-export const RoleModel: K8sKind = {
+export const RoleModel: K8sModel = {
   label: 'Role',
   // t('public~Role')
   labelKey: 'public~Role',
@@ -566,7 +566,7 @@ export const RoleModel: K8sKind = {
   labelPluralKey: 'public~Roles',
 };
 
-export const SelfSubjectAccessReviewModel: K8sKind = {
+export const SelfSubjectAccessReviewModel: K8sModel = {
   label: 'SelfSubjectAccessReview',
   // t('public~SelfSubjectAccessReview')
   labelKey: 'public~SelfSubjectAccessReview',
@@ -582,7 +582,7 @@ export const SelfSubjectAccessReviewModel: K8sKind = {
   labelPluralKey: 'public~SelfSubjectAccessReviews',
 };
 
-export const ResourceAccessReviewsModel: K8sKind = {
+export const ResourceAccessReviewsModel: K8sModel = {
   label: 'ResourceAccessReview',
   // t('public~ResourceAccessReview')
   labelKey: 'public~ResourceAccessReview',
@@ -598,7 +598,7 @@ export const ResourceAccessReviewsModel: K8sKind = {
   labelPluralKey: 'public~ResourceAccessReviews',
 };
 
-export const LocalResourceAccessReviewsModel: K8sKind = {
+export const LocalResourceAccessReviewsModel: K8sModel = {
   label: 'LocalResourceAccessReview',
   // t('public~LocalResourceAccessReview')
   labelKey: 'public~LocalResourceAccessReview',
@@ -614,7 +614,7 @@ export const LocalResourceAccessReviewsModel: K8sKind = {
   labelPluralKey: 'public~LocalResourceAccessReviews',
 };
 
-export const PersistentVolumeModel: K8sKind = {
+export const PersistentVolumeModel: K8sModel = {
   label: 'PersistentVolume',
   // t('public~PersistentVolume')
   labelKey: 'public~PersistentVolume',
@@ -628,7 +628,7 @@ export const PersistentVolumeModel: K8sKind = {
   labelPluralKey: 'public~PersistentVolumes',
 };
 
-export const PersistentVolumeClaimModel: K8sKind = {
+export const PersistentVolumeClaimModel: K8sModel = {
   label: 'PersistentVolumeClaim',
   // t('public~PersistentVolumeClaim')
   labelKey: 'public~PersistentVolumeClaim',
@@ -643,7 +643,7 @@ export const PersistentVolumeClaimModel: K8sKind = {
   labelPluralKey: 'public~PersistentVolumeClaims',
 };
 
-export const StatefulSetModel: K8sKind = {
+export const StatefulSetModel: K8sModel = {
   label: 'StatefulSet',
   // t('public~StatefulSet')
   labelKey: 'public~StatefulSet',
@@ -660,7 +660,7 @@ export const StatefulSetModel: K8sKind = {
   labelPluralKey: 'public~StatefulSets',
 };
 
-export const ResourceQuotaModel: K8sKind = {
+export const ResourceQuotaModel: K8sModel = {
   label: 'ResourceQuota',
   // t('public~ResourceQuota')
   labelKey: 'public~ResourceQuota',
@@ -675,7 +675,7 @@ export const ResourceQuotaModel: K8sKind = {
   labelPluralKey: 'public~ResourceQuotas',
 };
 
-export const ClusterResourceQuotaModel: K8sKind = {
+export const ClusterResourceQuotaModel: K8sModel = {
   label: 'ClusterResourceQuota',
   // t('public~ClusterResourceQuota')
   labelKey: 'public~ClusterResourceQuota',
@@ -692,7 +692,7 @@ export const ClusterResourceQuotaModel: K8sKind = {
   crd: true,
 };
 
-export const AppliedClusterResourceQuotaModel: K8sKind = {
+export const AppliedClusterResourceQuotaModel: K8sModel = {
   label: 'AppliedClusterResourceQuota',
   // t('public~AppliedClusterResourceQuota')
   labelKey: 'public~AppliedClusterResourceQuota',
@@ -709,7 +709,7 @@ export const AppliedClusterResourceQuotaModel: K8sKind = {
   crd: true,
 };
 
-export const NetworkPolicyModel: K8sKind = {
+export const NetworkPolicyModel: K8sModel = {
   label: 'NetworkPolicy',
   // t('public~NetworkPolicy')
   labelKey: 'public~NetworkPolicy',
@@ -725,7 +725,7 @@ export const NetworkPolicyModel: K8sKind = {
   id: 'networkpolicy',
 };
 
-export const CustomResourceDefinitionModel: K8sKind = {
+export const CustomResourceDefinitionModel: K8sModel = {
   label: 'CustomResourceDefinition',
   // t('public~CustomResourceDefinition')
   labelKey: 'public~CustomResourceDefinition',
@@ -741,7 +741,7 @@ export const CustomResourceDefinitionModel: K8sKind = {
   labelPluralKey: 'public~CustomResourceDefinitions',
 };
 
-export const CronJobModel: K8sKind = {
+export const CronJobModel: K8sModel = {
   label: 'CronJob',
   // t('public~CronJob')
   labelKey: 'public~CronJob',
@@ -758,7 +758,7 @@ export const CronJobModel: K8sKind = {
   propagationPolicy: 'Foreground',
 };
 
-export const StorageClassModel: K8sKind = {
+export const StorageClassModel: K8sModel = {
   label: 'StorageClass',
   // t('public~StorageClass')
   labelKey: 'public~StorageClass',
@@ -774,7 +774,7 @@ export const StorageClassModel: K8sKind = {
   id: 'storageclass',
 };
 
-export const LimitRangeModel: K8sKind = {
+export const LimitRangeModel: K8sModel = {
   label: 'LimitRange',
   // t('public~LimitRange')
   labelKey: 'public~LimitRange',
@@ -789,7 +789,7 @@ export const LimitRangeModel: K8sKind = {
   labelPluralKey: 'public~LimitRanges',
 };
 
-export const APIServiceModel: K8sKind = {
+export const APIServiceModel: K8sModel = {
   label: 'APIService',
   // t('public~APIService')
   labelKey: 'public~APIService',
@@ -806,7 +806,7 @@ export const APIServiceModel: K8sKind = {
   crd: true,
 };
 
-export const UserModel: K8sKind = {
+export const UserModel: K8sModel = {
   label: 'User',
   // t('public~User')
   labelKey: 'public~User',
@@ -823,7 +823,7 @@ export const UserModel: K8sKind = {
   crd: true,
 };
 
-export const GroupModel: K8sKind = {
+export const GroupModel: K8sModel = {
   label: 'Group',
   // t('public~Group')
   labelKey: 'public~Group',
@@ -842,7 +842,7 @@ export const GroupModel: K8sKind = {
 
 // Cluster API resources
 // https://github.com/openshift/cluster-api
-export const MachineModel: K8sKind = {
+export const MachineModel: K8sModel = {
   label: 'Machine',
   // t('public~Machine')
   labelKey: 'public~Machine',
@@ -859,7 +859,7 @@ export const MachineModel: K8sKind = {
   crd: true,
 };
 
-export const MachineSetModel: K8sKind = {
+export const MachineSetModel: K8sModel = {
   label: 'MachineSet',
   // t('public~MachineSet')
   labelKey: 'public~MachineSet',
@@ -876,7 +876,7 @@ export const MachineSetModel: K8sKind = {
   crd: true,
 };
 
-export const MachineDeploymentModel: K8sKind = {
+export const MachineDeploymentModel: K8sModel = {
   label: 'MachineDeployment',
   // t('public~MachineDeployment')
   labelKey: 'public~MachineDeployment',
@@ -893,7 +893,7 @@ export const MachineDeploymentModel: K8sKind = {
   crd: true,
 };
 
-export const MachineConfigPoolModel: K8sKind = {
+export const MachineConfigPoolModel: K8sModel = {
   label: 'MachineConfigPool',
   // t('public~MachineConfigPool')
   labelKey: 'public~MachineConfigPool',
@@ -910,7 +910,7 @@ export const MachineConfigPoolModel: K8sKind = {
   crd: true,
 };
 
-export const MachineConfigModel: K8sKind = {
+export const MachineConfigModel: K8sModel = {
   label: 'MachineConfig',
   // t('public~MachineConfig')
   labelKey: 'public~MachineConfig',
@@ -927,7 +927,7 @@ export const MachineConfigModel: K8sKind = {
   crd: true,
 };
 
-export const MachineAutoscalerModel: K8sKind = {
+export const MachineAutoscalerModel: K8sModel = {
   label: 'MachineAutoscaler',
   // t('public~MachineAutoscaler')
   labelKey: 'public~MachineAutoscaler',
@@ -944,7 +944,7 @@ export const MachineAutoscalerModel: K8sKind = {
   crd: true,
 };
 
-export const MachineHealthCheckModel: K8sKind = {
+export const MachineHealthCheckModel: K8sModel = {
   label: 'MachineHealthCheck',
   // t('public~MachineHealthCheck')
   labelKey: 'public~MachineHealthCheck',
@@ -962,7 +962,7 @@ export const MachineHealthCheckModel: K8sKind = {
 };
 
 // Openshift cluster resources
-export const ClusterOperatorModel: K8sKind = {
+export const ClusterOperatorModel: K8sModel = {
   label: 'ClusterOperator',
   // t('public~ClusterOperator')
   labelKey: 'public~ClusterOperator',
@@ -979,7 +979,7 @@ export const ClusterOperatorModel: K8sKind = {
   crd: true,
 };
 
-export const ClusterVersionModel: K8sKind = {
+export const ClusterVersionModel: K8sModel = {
   label: 'ClusterVersion',
   // t('public~ClusterVersion')
   labelKey: 'public~ClusterVersion',
@@ -996,7 +996,7 @@ export const ClusterVersionModel: K8sKind = {
   crd: true,
 };
 
-export const CSIDriverModel: K8sKind = {
+export const CSIDriverModel: K8sModel = {
   label: 'CSIDriver',
   // t('public~CSIDriver')
   labelKey: 'public~CSIDriver',
@@ -1013,7 +1013,7 @@ export const CSIDriverModel: K8sKind = {
   crd: true,
 };
 
-export const ClusterAutoscalerModel: K8sKind = {
+export const ClusterAutoscalerModel: K8sModel = {
   label: 'ClusterAutoscaler',
   // t('public~ClusterAutoscaler')
   labelKey: 'public~ClusterAutoscaler',
@@ -1031,7 +1031,7 @@ export const ClusterAutoscalerModel: K8sKind = {
 };
 
 // OpenShift global configuration
-export const OAuthModel: K8sKind = {
+export const OAuthModel: K8sModel = {
   label: 'OAuth',
   // t('public~OAuth')
   labelKey: 'public~OAuth',
@@ -1048,7 +1048,7 @@ export const OAuthModel: K8sKind = {
   crd: true,
 };
 
-export const InfrastructureModel: K8sKind = {
+export const InfrastructureModel: K8sModel = {
   label: 'Infrastructure',
   // t('public~Infrastructure')
   labelKey: 'public~Infrastructure',
@@ -1065,7 +1065,7 @@ export const InfrastructureModel: K8sKind = {
   crd: true,
 };
 
-export const NetworkOperatorConfigModel: K8sKind = {
+export const NetworkOperatorConfigModel: K8sModel = {
   label: 'Network',
   labelPlural: 'Networks',
   apiVersion: 'v1',
@@ -1078,7 +1078,7 @@ export const NetworkOperatorConfigModel: K8sKind = {
   crd: true,
 };
 
-export const ConsoleOperatorConfigModel: K8sKind = {
+export const ConsoleOperatorConfigModel: K8sModel = {
   label: 'Console',
   labelPlural: 'Consoles',
   apiVersion: 'v1',
@@ -1091,7 +1091,7 @@ export const ConsoleOperatorConfigModel: K8sKind = {
   crd: true,
 };
 
-export const ConsoleLinkModel: K8sKind = {
+export const ConsoleLinkModel: K8sModel = {
   label: 'ConsoleLink',
   // t('public~ConsoleLink')
   labelKey: 'public~ConsoleLink',
@@ -1108,7 +1108,7 @@ export const ConsoleLinkModel: K8sKind = {
   crd: true,
 };
 
-export const ConsoleCLIDownloadModel: K8sKind = {
+export const ConsoleCLIDownloadModel: K8sModel = {
   label: 'ConsoleCLIDownload',
   // t('public~ConsoleCLIDownload')
   labelKey: 'public~ConsoleCLIDownload',
@@ -1125,7 +1125,7 @@ export const ConsoleCLIDownloadModel: K8sKind = {
   crd: true,
 };
 
-export const ConsoleNotificationModel: K8sKind = {
+export const ConsoleNotificationModel: K8sModel = {
   label: 'ConsoleNotification',
   // t('public~ConsoleNotification')
   labelKey: 'public~ConsoleNotification',
@@ -1142,7 +1142,7 @@ export const ConsoleNotificationModel: K8sKind = {
   crd: true,
 };
 
-export const ConsoleExternalLogLinkModel: K8sKind = {
+export const ConsoleExternalLogLinkModel: K8sModel = {
   label: 'ConsoleExternalLogLink',
   // t('public~ConsoleExternalLogLink')
   labelKey: 'public~ConsoleExternalLogLink',
@@ -1159,7 +1159,7 @@ export const ConsoleExternalLogLinkModel: K8sKind = {
   crd: true,
 };
 
-export const ConsoleYAMLSampleModel: K8sKind = {
+export const ConsoleYAMLSampleModel: K8sModel = {
   label: 'ConsoleYAMLSample',
   // t('public~ConsoleYAMLSample')
   labelKey: 'public~ConsoleYAMLSample',
@@ -1176,7 +1176,7 @@ export const ConsoleYAMLSampleModel: K8sKind = {
   crd: true,
 };
 
-export const VolumeSnapshotModel: K8sKind = {
+export const VolumeSnapshotModel: K8sModel = {
   label: 'VolumeSnapshot',
   // t('public~VolumeSnapshot')
   labelKey: 'public~VolumeSnapshot',
@@ -1193,7 +1193,7 @@ export const VolumeSnapshotModel: K8sKind = {
   crd: true,
 };
 
-export const VolumeSnapshotClassModel: K8sKind = {
+export const VolumeSnapshotClassModel: K8sModel = {
   label: 'VolumeSnapshotClass',
   // t('public~VolumeSnapshotClass')
   labelKey: 'public~VolumeSnapshotClass',
@@ -1210,7 +1210,7 @@ export const VolumeSnapshotClassModel: K8sKind = {
   crd: true,
 };
 
-export const VolumeSnapshotContentModel: K8sKind = {
+export const VolumeSnapshotContentModel: K8sModel = {
   label: 'VolumeSnapshotContent',
   // t('public~VolumeSnapshotContent')
   labelKey: 'public~VolumeSnapshotContent',
@@ -1227,7 +1227,7 @@ export const VolumeSnapshotContentModel: K8sKind = {
   crd: true,
 };
 
-export const ConsolePluginModel: K8sKind = {
+export const ConsolePluginModel: K8sModel = {
   label: 'ConsolePlugin',
   // t('public~ConsolePlugin')
   labelKey: 'public~ConsolePlugin',
diff --git a/console/core/index.ts b/console/core/index.ts
index b75d9f7..36efaac 100644
--- a/console/core/index.ts
+++ b/console/core/index.ts
@@ -34,7 +34,7 @@ export const ServiceMonitorModel: K8sModel = {
   propagationPolicy: 'Foreground',
 };
 
-export const PodMonitorModel: K8sKind = {
+export const PodMonitorModel: K8sModel = {
   kind: 'PodMonitor',
   label: 'PodMonitor',
   // t('public~PodMonitor')
@@ -1261,7 +1261,7 @@ export const ConsolePluginModel: K8sModel = {
   crd: true,
 };
 
-export const CloudCredentialModel: K8sKind = {
+export const CloudCredentialModel: K8sModel = {
   kind: 'CloudCredential',
   label: 'CloudCredential',
   labelPlural: 'CloudCredentials',
@@ -1271,7 +1271,7 @@ export const CloudCredentialModel: K8sKind = {
   plural: 'cloudcredentials',
 };
 
-export const AuthenticationModel: K8sKind = {
+export const AuthenticationModel: K8sModel = {
   kind: 'Authentication',
   label: 'Authentication',
   labelPlural: 'Authentications',
