diff --git a/generated/kubevirt/release-0.49/models/K8sIoApimachineryPkgApisMetaV1ObjectMeta.ts b/generated/kubevirt/release-0.49/models/K8sIoApimachineryPkgApisMetaV1ObjectMeta.ts
index 93f6085..c286145 100644
--- a/generated/kubevirt/release-0.49/models/K8sIoApimachineryPkgApisMetaV1ObjectMeta.ts
+++ b/generated/kubevirt/release-0.49/models/K8sIoApimachineryPkgApisMetaV1ObjectMeta.ts
@@ -152,9 +152,9 @@ export function K8sIoApimachineryPkgApisMetaV1ObjectMetaFromJSONTyped(json: any,
         
         'annotations': !exists(json, 'annotations') ? undefined : json['annotations'],
         'clusterName': !exists(json, 'clusterName') ? undefined : json['clusterName'],
-        'creationTimestamp': !exists(json, 'creationTimestamp') ? undefined : (new Date(json['creationTimestamp'])),
+        'creationTimestamp': !exists(json, 'creationTimestamp') ? undefined : json['creationTimestamp'],
         'deletionGracePeriodSeconds': !exists(json, 'deletionGracePeriodSeconds') ? undefined : json['deletionGracePeriodSeconds'],
-        'deletionTimestamp': !exists(json, 'deletionTimestamp') ? undefined : (new Date(json['deletionTimestamp'])),
+        'deletionTimestamp': !exists(json, 'deletionTimestamp') ? undefined : json['deletionTimestamp'],
         'finalizers': !exists(json, 'finalizers') ? undefined : json['finalizers'],
         'generateName': !exists(json, 'generateName') ? undefined : json['generateName'],
         'generation': !exists(json, 'generation') ? undefined : json['generation'],
@@ -180,9 +180,9 @@ export function K8sIoApimachineryPkgApisMetaV1ObjectMetaToJSON(value?: K8sIoApim
         
         'annotations': value.annotations,
         'clusterName': value.clusterName,
-        'creationTimestamp': value.creationTimestamp === undefined ? undefined : (value.creationTimestamp.toISOString()),
+        'creationTimestamp': value.creationTimestamp === undefined ? undefined : value.creationTimestamp,
         'deletionGracePeriodSeconds': value.deletionGracePeriodSeconds,
-        'deletionTimestamp': value.deletionTimestamp === undefined ? undefined : (value.deletionTimestamp.toISOString()),
+        'deletionTimestamp': value.deletionTimestamp === undefined ? undefined : value.deletionTimestamp,
         'finalizers': value.finalizers,
         'generateName': value.generateName,
         'generation': value.generation,
diff --git a/kubevirt/models/K8sIoApimachineryPkgApisMetaV1ObjectMeta.ts b/kubevirt/models/K8sIoApimachineryPkgApisMetaV1ObjectMeta.ts
index 93f6085..c286145 100644
--- a/kubevirt/models/K8sIoApimachineryPkgApisMetaV1ObjectMeta.ts
+++ b/kubevirt/models/K8sIoApimachineryPkgApisMetaV1ObjectMeta.ts
@@ -152,9 +152,9 @@ export function K8sIoApimachineryPkgApisMetaV1ObjectMetaFromJSONTyped(json: any,
         
         'annotations': !exists(json, 'annotations') ? undefined : json['annotations'],
         'clusterName': !exists(json, 'clusterName') ? undefined : json['clusterName'],
-        'creationTimestamp': !exists(json, 'creationTimestamp') ? undefined : (new Date(json['creationTimestamp'])),
+        'creationTimestamp': !exists(json, 'creationTimestamp') ? undefined : json['creationTimestamp'],
         'deletionGracePeriodSeconds': !exists(json, 'deletionGracePeriodSeconds') ? undefined : json['deletionGracePeriodSeconds'],
-        'deletionTimestamp': !exists(json, 'deletionTimestamp') ? undefined : (new Date(json['deletionTimestamp'])),
+        'deletionTimestamp': !exists(json, 'deletionTimestamp') ? undefined : json['deletionTimestamp'],
         'finalizers': !exists(json, 'finalizers') ? undefined : json['finalizers'],
         'generateName': !exists(json, 'generateName') ? undefined : json['generateName'],
         'generation': !exists(json, 'generation') ? undefined : json['generation'],
@@ -180,9 +180,9 @@ export function K8sIoApimachineryPkgApisMetaV1ObjectMetaToJSON(value?: K8sIoApim
         
         'annotations': value.annotations,
         'clusterName': value.clusterName,
-        'creationTimestamp': value.creationTimestamp === undefined ? undefined : (value.creationTimestamp.toISOString()),
+        'creationTimestamp': value.creationTimestamp === undefined ? undefined : value.creationTimestamp,
         'deletionGracePeriodSeconds': value.deletionGracePeriodSeconds,
-        'deletionTimestamp': value.deletionTimestamp === undefined ? undefined : (value.deletionTimestamp.toISOString()),
+        'deletionTimestamp': value.deletionTimestamp === undefined ? undefined : value.deletionTimestamp,
         'finalizers': value.finalizers,
         'generateName': value.generateName,
         'generation': value.generation,
